app/views/user_mailer/email_changed.html.slim

```diff
@@ -12,4 +12,5 @@ table.hr
 
 p == t('.help',
       app: link_to(APP_NAME, Figaro.env.mailer_domain_name, class: 'gray'),
-      contact_phone_number: CONTACT_PHONE_NUMBER)
+      help_link: link_to(t('user_mailer.help_link_text'), MarketingSite.help_url),
+      contact_link: link_to(t('user_mailer.contact_link_text'), MarketingSite.contact_url))
```

Comment:
That would probably work, but to me the increased complexity outweighs the benefit of reducing duplication. These help text strings are similar but slightly different so to me it is clearer to just explicitly include them in each view. Does that work for you?